assets/helpers/tracking/googleTagManager.js

```diff
@@ -42,11 +42,11 @@ function getOrderId() {
 }
 
 function getContributionType() {
-  const param = getQueryParameter('contribType');
+  const param = getQueryParameter('selectedContributionType');
```

Comment:
do you think the param for the old flow should be named the same? it's basically determining the whole page, not just which tab is selected - so maybe its clearer to keep the old name

Reply:
Ok I'll remove. Can I get a conditional approve?